### PR TITLE
feat: heatmap divider

### PR DESCRIPTION
**Pull request by tihuan**
## Reason for Change

- #TICKET_NUMBER
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

1. Add divider

Screenshot 2023-09-05 at 8 15 25 AM

1. No divider in PNG/SVG

PNG:
Screenshot 2023-09-05 at 8 21 22 AM

SVG:
Screenshot 2023-09-05 at 8 15 16 AM

## Testing steps

- Either list QA steps or reasoning you feel QA is unnecessary
- Describe how you made sure to know that your changes worked. Should allow someone else to go verify your code without in depth knowledge.
- "Unit tested only", "tested in rdev by a, b, c, verifying feature worked by... ", "manually ran pipeline locally with these results: ..."

### CODE REVIEW

File: frontend/src/views/WheresMyGeneV2/components/HeatMap/style.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import styled from "@emotion/styled";
+import { gray300 } from "src/common/theme";
+import { X_AXIS_CHART_HEIGHT_PX } from "src/views/WheresMyGene/common/constants";
+import { Y_AXIS_CHART_WIDTH_PX } from "src/views/WheresMyGene/components/HeatMap/utils";
+
+enum ZIndex {
```
Comment (tihuan) [inline frontend/src/views/WheresMyGeneV2/components/HeatMap/style.ts:6]: Add enum to manage z-index order

File: frontend/src/views/WheresMyGeneV2/components/HeatMap/style.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  height: 100%;
+  position: absolute;
+  left: 24%;
+  width: 10px;
+  top: 0;
+  border-right: solid 0.5px ${gray300};
+  z-index: ${ZIndex.Divider};
+`;
+
+export const XAxisWrapper = styled.div`
```
Comment (tihuan) [inline frontend/src/views/WheresMyGeneV2/components/HeatMap/style.ts:22]: copied from the old file

File: frontend/src/views/WheresMyGeneV2/components/HeatMap/style.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  top: 0px;
+  left: 0px;
+  width: ${Y_AXIS_CHART_WIDTH_PX}px;
+  height: ${(props) => props.height || X_AXIS_CHART_HEIGHT_PX}px;
+  min-height: ${(props) => props.height}px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: end;
+`;
```
Comment (tihuan) [inline frontend/src/views/WheresMyGeneV2/components/HeatMap/style.ts:49]: copied from the old file

**Review by seve — APPROVED** [review]

**Review by seve — CHANGES_REQUESTED** [review]
looks like some commits from main were merged into this branch

**Comment by tihuan** [comment]
@seve I see that your branch is merged in `main`, so I just cherrypicked my commit on top of `main`!

**Review by atarashansky — APPROVED** [review]

**Pull request merged**